fetch.bs

```diff
@@ -2237,6 +2238,9 @@ the request.
   for HTTP/2, and equivalent information used to prioritize dispatch and processing of
   HTTP/1 fetches.
 
+ <li>
+  <p>If <var>request</var>'s <a for=request>destination</a> is
+  <code>fetch</code>, set it to be the empty string.
```

Comment:
then set it to the empty string*

(plus then, minus be)

Also, this can be on the same line as the `<li>` (apart from 100 columns wrapping) since it's the only child.